fdbclient/include/fdbclient/CommitProxyInterface.h

```diff
@@ -552,4 +552,30 @@ struct ExclusionSafetyCheckRequest {
 	}
 };
 
+struct GlobalConfigRefreshReply {
+	constexpr static FileIdentifier file_identifier = 12680327;
+	Arena arena;
+	RangeResultRef result;
+
+	GlobalConfigRefreshReply() {}
+	GlobalConfigRefreshReply(Arena const& arena, RangeResultRef result) : arena(arena), result(result) {}
+
+	template <class Ar>
+	void serialize(Ar& ar) {
+		serializer(ar, arena, result);
```

Comment:
Arena needs to be serialized last, I believe
```suggestion
		serializer(ar, result, arena);
```